cpp/rss_aggregator.cc

```diff
@@ -37,21 +38,23 @@
 #include "SyndicationFormat.h"
 #include "UBTools.h"
 #include "util.h"
+#include "VuFind.h"
 #include "XmlWriter.h"
 
 
 namespace {
 
 
 [[noreturn]] void Usage() {
-    ::Usage("[--process-name=new_process_name] config_file_path email_address xml_output_path");
+    ::Usage("[--use-web-proxy] subsystem_type email_address xml_output_path\n"
+            "where subsystem_type must be one of {ixtheo,relbib,krimdok}");
 }
 
 
-// These must be in sync with the sizes in data/ub_tools.sql (rss_aggregator table)
-const size_t MAX_ITEM_ID_LENGTH(100);
-const size_t MAX_ITEM_URL_LENGTH(512);
-const size_t MAX_ITEM_TITLE_LENGTH(200);
+// These must be in sync with the sizes in the vufind.rss_aggregator table!
```

Comment:
The correct new name is vufind.tuefind_rss_items

Reply:
Done